core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -572,12 +573,15 @@ public void start(Boolean isPrimary) throws IOException {
         if (inode.getPersistenceState() != PersistenceState.TO_BE_PERSISTED) {
           continue;
         }
+        if (inode.asFile().getShouldPersistTime() == Constants.NO_AUTO_PERSIST_VALUE) {
```

Comment:
nit: instead of a separate conditional, could we just add an `||` condition to the last statement?

Reply:
Changed, thanks